definitions/features/hammer.rb

```diff
@@ -197,11 +197,6 @@ def load_from_file(file_path)
     end
   end
 
-  def password_from_answers
-    return nil unless feature(:installer)
```

Comment:
That's good you encapsulated logic inside :installer feature.
In your changes, nil handling is missing.

Reply:
@kgaikwad add nil handling back, and reverted changes of e1369ac